tests/rule_based_profiler/parameter_builder/test_histogram_single_batch_parameter_builder.py

```diff
@@ -136,6 +136,59 @@ def test_histogram_single_batch_parameter_builder_alice_null_bins(
 
     assert parameter_container.parameter_nodes is None
 
+    with mock.patch(
+        "great_expectations.expectations.metrics.column_aggregate_metrics.column_partition._get_column_partition_using_metrics",
+        return_value=None,
+    ):
+        with pytest.raises(ge_exceptions.ProfilerExecutionError) as excinfo:
+            variables: Optional[ParameterContainer] = None
+            # noinspection PyUnusedLocal
+            parameter_builder.build_parameters(
+                domain=domain,
+                variables=variables,
+                parameters=parameters,
+                batch_request=batch_request,
+            )
+
+        assert (
+            "Partitioning values for HistogramSingleBatchParameterBuilder by column_partition_metric_single_batch_parameter_builder into bins encountered empty or non-existent elements."
+            in str(excinfo.value)
+        )
+
+
+@pytest.mark.integration
+def test_histogram_single_batch_parameter_builder_alice_nan_valued_bins(
+    alice_columnar_table_single_batch_context,
+):
+    data_context: DataContext = alice_columnar_table_single_batch_context
+
+    batch_request: dict = {
+        "datasource_name": "alice_columnar_table_single_batch_datasource",
+        "data_connector_name": "alice_columnar_table_single_batch_data_connector",
+        "data_asset_name": "alice_columnar_table_single_batch_data_asset",
+    }
+
+    parameter_builder: ParameterBuilder = HistogramSingleBatchParameterBuilder(
+        name="my_name",
+        bins="auto",
+        evaluation_parameter_builder_configs=None,
+        data_context=data_context,
+    )
+
+    metric_domain_kwargs: dict = {"column": "my_column"}
```

Comment:
Is this a real column or a dummy? Why does this produce `NaN` bins?

Reply:
@NathanFarmer I mocked it to produce `NaN` values -- as in the below.